.github/workflows/ci.yaml

```diff
@@ -103,6 +103,12 @@ jobs:
         if: matrix.platform == 'ubuntu-latest'
         run: |
           make build
+      - name: Write creds to files for render-helm-chart tests
+        if: matrix.platform == 'ubuntu-latest'
+        run: |
+          echo -e 'username=${{ secrets.RENDER_HELM_CHART_BOT_TOKEN }}\npassword=${{ secrets.RENDER_HELM_CHART_BOT_TOKEN }}' \
```

Comment:
If a contributor run the tests locally, will all tests pass?

Reply:
No. Is there a good way to mark these directories as unable to be tested locally? 

In kustomize, we gate such tests with an environment variable that we set in CI. I could do something similar here if desired.

Reply:
> Is there a good way to mark these directories as unable to be tested locally?

The test harness doesn't support it now. Currently we only have `skip`: https://github.com/GoogleContainerTools/kpt/blob/main/pkg/test/runner/README.md

Here is one option: You can add another config field maybe called `skipLocal` in the test harness in the kpt repo. When a developer test is locally, these tests will be skipped. When tests is run in the GH Action, we can use another env var to make all tests (including the ones with `skipLocal: true`) are tested.

If you have better solutions, please feel free to use.

Reply:
Thanks! Yeah that's what I was thinking too. I created a PR to add this option in the kpt repo: https://github.com/GoogleContainerTools/kpt/pull/3295